### PR TITLE
Json participation storage

### LINKED ISSUE #117: Add Participation json storage and loading

### PR DESCRIPTION
**Pull request by BlazeChron**
Stores Participation object in json file.
Closes #117

### CODE REVIEW

**Review by jinhanjinhan — APPROVED** [review]
Everything looks great!

File: src/main/java/seedu/address/logic/commands/EnrollCommand.java [hunk 1]
```diff
@@ -67,7 +68,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
         Tutorial tutorial = optionalTutorial.get();
 
-        Participation p = new seedu.address.model.participation.Participation(student, tutorial);
+        Participation p = new seedu.address.model.participation.Participation(student, tutorial, new ArrayList<>());
```
[the remark this answers is not in the document]
Comment (BlazeChron) [reply src/main/java/seedu/address/logic/commands/EnrollCommand.java:71]: I think that would be a good idea.

[the remark this answers is not in the document]
Comment (jinhanjinhan) [reply src/main/java/seedu/address/logic/commands/EnrollCommand.java:71]: yea just a thought, but i think this way works fine too.

**Review by gladwintan — APPROVED** [review]
LGTM

**Pull request merged**